### ISSUE TITLE
Лимит одновременных запросов

### ISSUE THREAD

**Issue by aksenov** [lensgolda/bykwords-search#2]
Текущая реализация лимита одновременных запросов скорее всего будет стопориться в случае, если застопорится один из запросов. Так как следующая тройка не пойдет в обработку, пока не будет закончена обработка текущей.

**Comment by lensgolda** [comment]
Согласен, сейчас это работает параллельно, но не асинхронно. Это я исправлю.

**Comment by aksenov** [comment]
Отлично. Буду ждать.
